types/mirrorx/index.d.ts

```diff
@@ -0,0 +1,219 @@
+// Type definitions for mirrorjs 
+// Project: https://github.com/mirrorjs/mirror
+// Definitions by: Aaronphy <https://github.com/aaronphy>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
```

Comment:
Do you need 2.4 here? Looks like react's minimum is 2.3 so you should be able to get away with 2.3?

Reply:
because  react-redux `s mininum is 2.4.